addon/mixins/osf-model.js

```diff
@@ -7,5 +7,37 @@ import DS from 'ember-data';
 
 export default Ember.Mixin.create({
     links: DS.attr('links'),
-    embeds: DS.attr('embed')
+    embeds: DS.attr('embed'),
+
+    relationshipLinks: DS.attr('links'),
```

Comment:
This seems to be the same content as appears in `links`- prefer one clear field of choice for a piece of data when possible.